market/connector/sftp/README.md

```diff
@@ -0,0 +1,81 @@
+**SFTP Connector** is an SFTP client which uses the SFTP protocol to transfer files securely to and from a remote computer.
+SFTP Connector is based on JSch [http://www.jcraft.com/jsch/] to handle the SFTP Connections and Operations.
```

Comment:
May I suggest some other wording to give this intro text a marketing flair?

Suggestion:
Axon Ivy's SFTP Connector helps you to accelerate process automation initiatives by integrating secure file transfer into your process work. With this SFTP client you can easily transfer files securely to and from a remote computer. This connector: 
- uses the SFTP protocol
- is based on JSch [http://www.jcraft.com/jsch/] to handle the SFTP Connections and Operations
- is a pure Java implementation of SSH2
- allows you to connect to an sshd server and use port forwarding, X11 forwarding, file transfer, etc.
- makes it easy to integrate secure file transfer into your process work

Reply:
Updated accordingly ;)